app/assets/javascripts/components/Columns/EpicColumn.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Column from './ColumnItem';
+import Stories from '../stories/Stories';
+import PropTypes from "prop-types";
+import StoryPropTypes from '../shapes/story';
+import { totalPoints, donePoints, remainingPoints } from '../../models/beta/story';
+import { EPIC } from '../../models/beta/column';
+import { receiveStories } from '../../actions/story';
+import { connect } from "react-redux";
+
+export const EpicColumnHeader = ({ stories }) =>
+  <div className="Sprint__header">
+    {I18n.t('projects.show.done')}: { donePoints(stories) } / {I18n.t('remaining')}: { remainingPoints(stories) }
+
+    <span className="done-points">{I18n.t('projects.reports.points')}: { totalPoints(stories) }</span>
+  </div>
+
+const EpicColumn = ({ stories, receiveStories }) => (
+  <Column
+    onClose={() => receiveStories([], EPIC)}
```

Comment:
Instead of that, let's create a `CLOSE_EPIC_COLUMN` that sets the EPIC scope to be an empty array inside of the reducer instead of putting this responsibility on the component. We did something similar to the search https://github.com/Codeminer42/cm42-central/blob/82df013348b9945e624c414dfcdc856fef750eb6/app/assets/javascripts/reducers/projectBoard.js#L44-L51